cmp/form/FormModel.ts

```diff
@@ -22,13 +43,23 @@ export interface FormConfig {
     /** Map of initial values for fields in this model. */
     initialValues?: PlainObject;
 
+    /** Options governing persistence of the form state. */
+    persistWith?: FormPersistOptions;
+
     disabled?: boolean;
     readonly?: boolean;
 
     /** @internal */
     xhImpl?: boolean;
 }
 
+export interface FormPersistOptions extends PersistOptions {
```

Comment:
Should this be a waterfall where only one of these two properties is respected?

i.e. 
`persistedFields = fieldsToExclude ? without(allFields, ...fieldsToExclude) : fieldsToInclude ?? allFields`

Just thinking it would be strange to specify both since it would be clearer to just not include the "excluded" fields in your list of "included" fields.